chapters/lexicalstructure.tex

```diff
@@ -247,6 +247,24 @@ \subsection{Strings}\label{strings}
 \end{lstlisting}
 \end{nonnormative}
 
+\subsection{Units of Literal Constants}\label{units-literal-constants}
+
+Literal \lstinline!Real! and \lstinline!Integer! constants are implicitly assumed to have \lstinline!unit = "1"! if they appear in expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string.
```

Comment:
```suggestion
Literal \lstinline!Real! and \lstinline!Integer! constants are implicitly assumed to have a dimensionless unit if they appear in expressions containing variables, parameters, or constants that have a non-empty \lstinline!unit! attribute string, or function calls that have a non-empty \lstinline!unit! attribute of the output variable. The same rule applies to function call inputs.
```

Reply:
Here I followed Hans's suggestion to use "dimensionless unit" as in built-in functions also in the rule about expressions. I also expanded the text to clarify how to deal with function calls in expressions.

Reply:
Seems ok.